h/db/__init__.py

```diff
@@ -103,7 +97,7 @@ def close_the_sqlalchemy_session(_request):
     return session
 
 
-def _maybe_create_default_organization(engine, authority):
+def _maybe_create_default_organization(engine, authority):  # pragma: no cover
```

Comment:
This now gets called _before_ the tests, when `tox.ini` runs `init_db` before running `pytest`. So it no longer gets hit by the test coverage. (Previously this was called by a test fixture so it was covered.)

I could add a unit test for this but :man_shrugging: